xapian-bindings/java/SmokeTest.java

```diff
@@ -121,8 +121,20 @@ public static void main(String[] args) throws Exception {
 		System.exit(1);
 	    }
 	    Enquire enq = new Enquire(db);
-	    enq.setQuery(new Query(Query.OP_OR, "there", "is"));
+	    enq.setQuery(new Query("dog"));
 	    MSet mset = enq.getMSet(0, 10);
+	    // MSet size is 0
+	    if (mset.size() == 0) {
+	    for (MSetIterator.MatchDescriptor descriptor : mset) {
+	    System.err.println("Empty iterator entered invalid section (from MSet java iterator)");
```

Comment:
This alignment still looks wrong to me; l128 should be indented from l127, and l129 indented even further. What editor are you using? I'm wondering if it's possible to tweak some configuration there so it'll get this right for you automatically :-)